### PR TITLE
Fixed archive_events script & Created DemoPage availability flag

### PR DESCRIPTION
**Pull request by kave**
## Additions

- enabled demo_page use in flapjack

## Fixed

- Fixed archive_events script to run in production

## Testing

- Set `Debug=False` and verify you cannot create the `DemoPage`
 - *Note* run server in insecure mode after a `collectstatic`
 - `./cfgov/manage.py runserver --insecure`
- Set environment variable `export DEMO_PAGE=True` and verify you can create the `DemoPage`

## Review

- @kurtw 
- @richaagarwal 
- @rosskarchner 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

…emo_page in flapjack

### CODE REVIEW

File: cfgov/cfgov/settings/local.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from .base import *
 
 DEBUG = True
-INSTALLED_APPS += ('django_extensions', 'wagtail.contrib.wagtailstyleguide')
+INSTALLED_APPS += ('wagtail.contrib.wagtailstyleguide',)
```
Comment (anselmbradford) [inline cfgov/cfgov/settings/local.py]: Trailing comma is a python best practice, right?

Comment (rosskarchner) [inline cfgov/cfgov/settings/local.py]: In the case of a tuple, it's the only way to make a single-item tuple:

```
>>> ("orange")
'orange'
>>> ("orange",)
('orange',)
```

**Comment by richaagarwal** [comment]
:+1:

**Comment by rosskarchner** [comment]
:+1:

**Pull request merged**